src/App.tsx

```diff
@@ -7,24 +7,29 @@ import { IAlbum, ISong } from 'dxmp-common';
 import * as React from 'react';
 
 interface IState {
-  albums: IAlbum[],
+  albums: {[albumID: number]: IAlbum},
```

Comment:
You could do `Dictionary<IAlbum>` (`Dictionary` is part of `dxmp-common`) (blanket statement for similar occurrences)